### ISSUE TITLE
New terms for CITE-seq and 10x feature barcoding technologies

### ISSUE THREAD

**Issue by anjaf** [EBISPOT/efo#1092]
Also relating to the meeting with the ArrayExpress/Expression Atlas curation team.

To more accurately annotate sequencing libraries from CITE-seq and 10x multiplexing experiments that measure for example cell surface protein expression or use this for sample multiplexing, we require a few more defined "single cell library construction" terms.

New term label | Parent term(s) | Definition
-- | -- | --
10x scATAC-seq | "10x" http://www.ebi.ac.uk/efo/EFO_0008995; "scATAC-seq" http://www.ebi.ac.uk/efo/EFO_0010891 | A single cell ATAC-seq method for profiling of chromatin accessibility. For library construction single cells or single nuclei are isloated into droplets using 10x Chromium technnolgy.
CITE-seq (cell surface protein profiling) | "CITE-seq" http://www.ebi.ac.uk/efo/EFO_0009294 | The use of CITE-seq technology with the purpose of quantifying cell surface protein expression in addition to transcriptomic readout in single cells.
CITE-seq (sample multiplexing) | "CITE-seq" http://www.ebi.ac.uk/efo/EFO_0009294 | The use of CITE-seq technology with the purpose of mulitplexing single-cell samples from different sources.
10x feature profiling | "10x" http://www.ebi.ac.uk/efo/EFO_0008995 | 10x feature barcode technology is a method for adding extra channels of information to cells by running single-cell gene expression in parallel with other assays. This is done by generating additional sequencing libraries within the same droplet containing a single cell. The readout of these libraries are known oligonuceotides that are coupled to cell surface proteins antibodies or sgRNAs.
10x feature barcode (cell surface protein profiling) | "10x feature profiling" (new term) | The use of 10x Genomics feature barcode technology with the purpose of quantifying cell surface protein expression in addition to transcriptomic readout in single cells.
10x feature barcode (sample multiplexing) | "10x feature profiling" (new term) | The use of 10x Genomics feature barcode technology with the purpose of mulitplexing single-cell samples from different sources.
10x feature barcode (CRISPR screening) | "10x feature profiling" (new term) | The use of 10x Genomics feature barcode technology with the purpose of performing a CRISPR screening.

My ORCID is 0000-0002-8674-0039. 

Please let me know if you need any further information.

**Comment by paolaroncaglia** [comment]
Hi again @anjaf ,
Would you be able to suggest PMIDs or other reliable references for the new term requests above? Not strictly required, but nice to have. Ideally papers or reports describing the technology, but studies that just use (cite) the technology would also work. Other than that, the suggested new terms look good to me. Thanks.

**Comment by anjaf** [comment]
Sure, for the 2 CITE-seq terms we can reference [PMID: 30567574](https://pubmed.ncbi.nlm.nih.gov/30567574/)
I'm not sure if 10x has published their methods in peer-review journal but I'll check and otherwise find some usage examples.

**Comment by paolaroncaglia** [comment]
Hi Anja,
The new terms you requested will be available after tomorrow's EFO release:

EFO_0030007 (10x scATAC-seq)
EFO_0030008 (CITE-seq (cell surface protein profiling))
EFO_0030009 (CITE-seq (sample multiplexing))
EFO_0030010 (10x feature profiling)
EFO_0030011 (10x feature barcode (cell surface protein profiling))
EFO_0030012 (10x feature barcode (sample multiplexing))
EFO_0030013 (10x feature barcode (CRISPR screening))

Best,
Paola
